lib/Saml2/Metadata.php

```diff
@@ -96,6 +97,42 @@ public static function builder($sp, $authnsign = false, $wsign = false, $validUn
             $strContacts = "\n".implode("\n", $contactsInfo);
         }
 
+        $strAttributeConsumingService = '';
+        if (isset($sp['attributeConsumingService'])) {
+            $attrCsDesc = '';
+            if (isset($sp['attributeConsumingService']['serviceDescription'])) {
+                $attrCsDesc = sprintf(
+                    '            <md:ServiceDescription xml:lang="en">%s</md:ServiceDescription>' . PHP_EOL,
+                    $sp['attributeConsumingService']['serviceDescription']
+                );
+            }
+            if (!isset($sp['attributeConsumingService']['serviceName'])) {
+                $sp['attributeConsumingService']['serviceName'] = 'Service';
+            }
+            $requestedAttributeData = [];
```

Comment:
can you use array() instead of []
Travis fails on php5.3:
```
Parse error: syntax error, unexpected '[' in /home/travis/build/onelogin/php-saml/lib/Saml2/Metadata.php on line 112
```